### ISSUE TITLE
Delete song confirmation. Please remove it!!!

### ISSUE THREAD

**Issue by uq1** [ComputerElite/BM#29]
Please, PLEASE, remove this "are you sure" (if I wasn't I wouldn't have clicked it!!!) popup for every song. I'm trying to delete hundreds of songs, asking for every song is just insane. There's no "delete all" button.

BMBF can select multiple files, but there's no "delete selected" option. Deleting custom songs should not be this difficult.

How is this "delete all" not a thing? Surely everyone here wants to delete all the junk every so often.

Thankyou so much for giving the community a playlist editor that actually works, managing quest 2 beatsaber is a nightmare.

**Comment by ComputerElite** [comment]
I didn't expect someone to still use this program. And even lees I thought it'd still work. If the code isn't too bad I'll see if I can update the program

**Comment by ComputerElite** [comment]
Okay, as I did not feel comfortable at all modifying the code largely I simply made the `are you sure to delete song x` prompt only appear on the first song if you are deleting a bunch of them (you can select multiple songs with ctrl + click and shift + click iirc). I hope that satisfies your needs

**Comment by uq1** [comment]
Hehe. Yeah, I had to search for hours just to find some software with a delete button at all.

There's a lot of playlist managers, but none of them seem to manage the actual files, just the playlists. Your software was the only one I could find that actually had the functionality.

Your checkbox is the perfect solution. Thankyou so much.

I actually cloned the code and was going to try to remove the popup myself, but unfortunately I only have VS 2017 and 2021, which lack the old system.json functionality you used (yay C# and microsoft for constantly removing things).

I ended up placing an object on my 'y' key and left it running, hahaha!

I don't know if you have the time these days, or can be bothered, but there really is a lack of good management software for Q2 BeatSaber. Your software has functionality that just doesn't seem to be available elsewhere, it may be worth working on further, if you have the time.

As an open source dev myself, I know how much work we put in, usually for little appreciation, so I thank you so very much for the time you have put in to your software (and for adding this functionality so quickly!!!). I am sure I will need it again. I make a bs [automapper](https://github.com/uq1/BSAutoGenerator) (if you try it, use the latest release in the code's builds folder, not the older github builds - the new BeatSage remapper is pretty cool), and I need to replace my whole custom songs list often for updates, but hopefully it will also be of use to others in the future.
